src/routes.tsx

```diff
@@ -0,0 +1,60 @@
+import ServiceGraphRouteHandler from './pages/ServiceGraph/ServiceGraphRouteHandler';
+import ServiceListPage from './pages/ServiceList/ServiceListPage';
+import IstioConfigPage from './pages/IstioConfigList/IstioConfigListPage';
+import ServiceJaegerPage from './pages/ServiceJaeger/ServiceJaegerPage';
+import ServiceDetailsPage from './pages/ServiceDetails/ServiceDetailsPage';
+import IstioConfigDetailsPage from './pages/IstioConfigDetails/IstioConfigDetailsPage';
+
+const baseName = '/';
+
+/**
+ * Return array of objects that describe vertical menu
+ * @return {array}
+ */
+const routes = () => [
+  {
+    iconClass: 'fa pficon-topology',
+    title: 'Graph',
+    to: '/service-graph/all',
+    redirect: true,
+    component: ServiceGraphRouteHandler,
+    pathsActive: [/\/service-graph\/(.*)\//g]
+  },
+  {
+    iconClass: 'fa pficon-service',
+    title: 'Services',
+    to: '/services',
+    component: ServiceListPage,
+    pathsActive: [/\/namespaces\/(.*)\/services\/(.*)/g]
```

Comment:
Hmm... All these regex should start with `^` in order to work like the old code (that is using `startsWith`). Also, I think that the `g` flag shouldn't be used.

Reply:
+1 
Using `^` will improve performance as well.

Reply:
Changed